public/js/components/AtomCreate/AtomCreateTypeSelect.js

```diff
@@ -1,22 +1,38 @@
 import React from 'react';
 import {Link} from 'react-router';
 
-import {allAtomTypes} from '../../constants/atomData';
+import {workshopEditableAtomTypes, getNonEditableAtomTypes} from '../../constants/atomData';
 import {AtomTypeCard} from '../AtomTypeCard/AtomTypeCard';
 
 
 export class AtomCreateTypeSelect extends React.Component {
 
+  renderAtomType(atomType) {
+    return (
+      <Link
+        to={`/create/${atomType.type}`}
+        className={"create__link"}
+        key={atomType.type}
+        >
+        <AtomTypeCard atomType={atomType} />
+      </Link>
+    )
+  }
+
   render () {
     return (
       <div className="page__section">
         <h1 className="page__subheading">Create an Atom</h1>
         <div className="create__cards">
-          {allAtomTypes.map((atomType) => (
-            <Link to={`/create/${atomType.type}`} className="create__link" key={atomType.type}>
-              <AtomTypeCard atomType={atomType} />
-            </Link>
-          ))}
+          {workshopEditableAtomTypes.map(this.renderAtomType)}
+        </div>
+        <h1 className="page__subheading">Other Atoms</h1>
+        <div className="create__note">
```

Comment:
I think this part should be in a separate `page__section` so we get borders and spacing.